### PR TITLE
[WIP] Upgrade to django_filter v2

### LINKED ISSUE #136: Upgrade to django-filter v2

### PR DESCRIPTION
**Pull request by DavisRayM**
- [x] Remove Backports Library
- [x] Drop Python2 and Python3.5 Support
- [x] Update Tests
- [x] Update Modules ( Models, Serializers, Viewsets, etc.)
- [ ] Update Documentation

Fix #136

### CODE REVIEW

**Comment by moshthepitt** [comment]
Also need to remove:

- future
- six
- and change any code that may have used them

File: tasking/filters.py [hunk 1]
```diff
@@ -40,21 +39,21 @@ class TaskFilterSet(filters.FilterSet):
...
+        field_name='date',
+        field_class=forms.DateField,
+        lookup_choices=DATETIME_LOOKUPS,
+        method='filter_timing')
+    start_time = filters.LookupChoiceFilter(
+        field_name='start_time',
+        field_class=forms.TimeField,
+        lookup_choices=TIME_LOOKUPS,
+        method='filter_timing')
+    end_time = filters.LookupChoiceFilter(
```
Comment (DavisRayM) [inline tasking/filters.py]: The Usage of this changes how api requests should be made. With the use of a LookupChoiceFilter request will now look like this: `https://example.com/api/v1/task/?end_time=example_time&end_time_lookup=exact`

Comment (DavisRayM) [reply tasking/filters.py]: Currently looking onto ways to change the filterset to work with the usual `https://example.com/api/v1/task/?end_time_gt=exampletime`.

**Comment by DavisRayM** [comment]
Can't seem to find any sane way to bring back the old way of doing the requests : `https://example.com/api/v1/task/?end_time__gt=exampletime`. Any ideas and help on this would be greatly appreciated.

**Pull request closed without merging**